docs-src/docs/guides/walkthrough.md

```diff
@@ -0,0 +1,317 @@
+* `guestbook-svc`: a name for the service you are creating
+* `guestbook-dep`: the name of the deploy that you created in Part 3
+
+
+---
+
+### Check the Status of the Service
+
+After a service has been created, it may take several minutes for that service to completely finish deploying. A service's status may be checked by using the `service get` command.
+
+Let's take a peek at our **guestbook-svc** service. At the command prompt, execute the following:
+
+`l0 service get demo-env:guestbook-svc`
+
+If we're quick enough, we'll be able to see the first stage of the process (this is what was output after running the `service create` command up in Part 4). We should see an asterisk (\*) next to the name of the **guestbook-dep:1** deploy, which indicates that the service is in a transitional state:
+```
+SERVICE ID    SERVICE NAME   ENVIRONMENT  LOADBALANCER  DEPLOYMENTS       SCALE
+guestbo9364b  guestbook-svc  demo-env     guestbook-lb  guestbook-dep:1*  0/1
+```
+
+In the next phase of deployment, if we execute the `service get` command again, we will see **(1)** in the **Scale** column; this indicates that 1 copy of the service is transitioning to an active state:
+```
+SERVICE ID    SERVICE NAME   ENVIRONMENT  LOADBALANCER  DEPLOYMENTS       SCALE
+guestbo9364b  guestbook-svc  demo-env     guestbook-lb  guestbook-dep:1*  0/1 (1)
+```
+
+In the final phase of deployment, we should see that the service has been fully scaled up to the desired count. If we execute the `service get` command again, you will see the following output:
+
+```
+SERVICE ID    SERVICE NAME   ENVIRONMENT  LOADBALANCER  DEPLOYMENTS       SCALE
+guestbo9364b  guestbook-svc  demo-env     guestbook-lb  guestbook-dep:1   1/1
+```
+
+
+---
+
+### Get the Application's URL
+
+Once the service has been completely deployed, we can obtain the URL for the application and launch it in a browser.
+
+At the command prompt, execute the following:
+
+`l0 loadbalancer get demo-env:guestbook-lb`
+
+This will give us the following output:
+
+```
+LOADBALANCER ID  LOADBALANCER NAME  ENVIRONMENT  SERVICE        PORTS       PUBLIC  URL
+guestbodb65a     guestbook-lb       demo-env     guestbook-svc  80:80/HTTP  true    <url>
+```
+
+Copy the value shown in the **URL** column and paste it into a web browser. The guestbook application will appear (once the service has completely finished deploying).
+
+
+---
+
+### Cleanup
+
+If you're finished with the example and don't want to continue with this walkthrough, you can instruct Layer0 to delete the environment and terminate the application and the Layer0 the application was using.
+
+`l0 environment delete demo-env`
+
+However, if you intend to continue through [Deployment 2](#deployment-2-guestbook-redis), you will want to keep the resources you made in this section.
+
+
+---
+
+## 1b: Deploy with Terraform
+
+Instead of using the Layer0 CLI directly, you can instead use the Layer0 Provider for Terraform, and deploy using Terraform. 
```

Comment:
Would be good to provide a link to _find out more about Terraform here_ for someone who is reading this without any prior knowledge of Terraform.